ML.md

```diff
@@ -0,0 +1,96 @@
+# Machine Learning - An overview
+
+Machine Learning (ML) is defined as the use of algorithms and computational statistics to learn from data without being explicitly programmed. It is a subsection of the artificial intelligence domain within computer science. The term Machine Learning was first coined in 1959. Recent rise in ML is due to availability of large number of dataset, and increased computation power
+
+
+## Traditional Programming
+
+
+Most of you may have already be familiar with traditional programming, where you start with a goal, write logical rules, and refine through testing until it works the way you want it to. Example of tradition programming are adding two given numbers. traditional programing is also called as Rule based approach
+
+
+<p align="center">
+  <img src="rulebased.png" alt="Rule Based"/>
+</p>
+
+## Machine Learning
+
+But there are certain problems which cannot be solve by traditional programming, for example consider creating an application which classify doodles made by kinder garden kids. Using traditional programming how would you differentiate between a cat and a human? you might say cat have a tail, whiskers, four legs etc.., looks simple and intutive right? now differentiate between a cat and dog? both have four legs, and tail which makes it little difficult to differentiate. Now Imagine having hundreds of such classes how many rules should you check. To tackle such problem we use Machine Learning, ML is all about learning from examples.
+
+<p align="center">
+  <img src="ml.png" alt="ML"/>
+</p>
+
+**Machine Learning is a specific field of Artificial Intelligence, where a system learns to find patterns in examples in order to make predictions.**
+
+
+In [Quick Draw](https://quickdraw.withgoogle.com/) **goal** is to create a model which classify given doodle into one of 345 predefined categories, Quick Draw Dataset is a collection of 50 million drawings across 345 categories. First we generate a hypthesis then update it by training
+
+
+### Flow
+
+1. We first understand user requirements
+2. Define an objective (Formulating problem)
+3. Collect data for training and evaluating
+4. learning algorthim generate a hypothesis function
+5. Hypothesis is used to predict on validation data if results are not satisfactory we try to enhance our hypothesis
+
+<p align="center">
```

Comment:
@Saichethan  please refer to using .md (markdown) for
the syntax for that is :
![](image) 

Also going to include a brief YT video that really helped me with .md formatting. 
https://www.youtube.com/watch?v=HUBNt18RFbo

Also, make sure you are following new submission instructions as well - as all images will be placed within the same branch.
https://github.com/section-io/engineering-education/blob/master/UPLOAD_INSTRUCTIONS.md

Reply:
HTML and other languages can be included as examples in code blocks if needed of course